routes/runtime/github.rb

```diff
@@ -78,6 +78,12 @@ class CloverRuntime
           fail CloverError.new(400, "InvalidRequest", "No workflow job data available")
         end
 
+        entry = repository.cache_entries_dataset
+          .exclude(committed_at: nil)
+          .where(key: key, version: version, scope: scope).all
+
+        fail CloverError.new(409, "AlreadyExists", "Scope: #{scope}, Key: #{key}, Version: #{version}") if entry.any?
```

Comment:
Do we end up at `else` branch?

https://github.com/actions/toolkit/blob/6c4e082c181a51609197e536ef5255a0c9baeef7/packages/cache/src/cache.ts#L224C1-L238C1

Reply:
Yes